pillar.example

```diff
@@ -385,3 +385,9 @@ apache:
     SSLProtocol: all -SSLv2 -SSLv3 -TLSv1
     SSLHonorCipherOrder: 'On'
     SSLOptions: "+StrictRequire"
+  server_status_access:
+    ip:
+      - 10.8.8.0/24
+    host:
+      - foo.example.com
+
```

Comment:
This blank line needs to be removed to pass the `yamllint` check:

* https://travis-ci.com/saltstack-formulas/apache-formula/jobs/268961933#L363-L365